File: Difficulty: Easy/Transpose of Matrix/transpose-of-matrix.py
class Solution:
    def transpose(self, mat):
        # code here
        rows = len(mat)
        cols = len(mat[0])
        
        for row in range(rows-1):
            for col in range(row+1, cols):
                mat[row][col], mat[col][row] = mat[col][row], mat[row][col]
            
        return mat
        

File: Difficulty: Easy/Transpose of Matrix/transpose-of-matrix.cpp
class Solution {
  public:
    vector<vector<int>> transpose(vector<vector<int>>& mat) {
        // code here
        int rows = mat.size();
        int cols = mat[0].size();

        for(int row=0;row<rows-1;row++)
        {
            for(int col=row+1;col<cols;col++)
            {
                swap(mat[row][col],mat[col][row]);
            }
        }
        return mat;
    }
};